src/main/java/mekanism/common/recipe/machines/CombinerRecipe.java

```diff
@@ -1,20 +1,26 @@
 package mekanism.common.recipe.machines;
 
-import mekanism.common.MekanismFluids;
-import mekanism.common.recipe.inputs.AdvancedMachineInput;
+import mekanism.common.recipe.inputs.DoubleMachineInput;
 import mekanism.common.recipe.outputs.ItemStackOutput;
+import net.minecraft.init.Blocks;
 import net.minecraft.item.ItemStack;
 
-public class CombinerRecipe extends AdvancedMachineRecipe<CombinerRecipe>
+public class CombinerRecipe extends DoubleMachineRecipe<CombinerRecipe>
 {
-	public CombinerRecipe(AdvancedMachineInput input, ItemStackOutput output)
+	public CombinerRecipe(DoubleMachineInput input, ItemStackOutput output)
 	{
 		super(input, output);
 	}
 
+	public CombinerRecipe(ItemStack input, ItemStack extra, ItemStack output)
+	{
+		super(input, extra, output);
+	}
+
+	@Deprecated // TODO remove this method in MC 1.13
```

Comment:
Why remove it? That would be a convenience method for the default.

Reply:
It seems rather arbitrary for cobblestone to be the default, and there's no way we'd add this method if we were building the combiner like this from scratch today.

Reply:
> arbitrary 

I disagree, given that 99% of ores are made from cobble/stone

Reply:
We don't have convenience methods for any other recipes, though. It just doesn't seem natural to have one here. (But if this is really a big deal, I am willing to un-deprecate them to get this merged though.)

Reply:
I'm fine with you suggesting it, but putting it in without mentioning it comes across as "we must do this"

To me this constructor defines what the default is

Reply:
Fair point; I'll reword the comment a bit.